registration-processor-default.properties

```diff
@@ -292,14 +292,15 @@ registration.processor.pause.packets.for.backpressure=true
 # The elapse time (in sec) beyond which the rids will be considered for reprocessing
 registration.processor.reprocess.elapse.time=300
```

Comment:
we have give the 4 hours here, the other "registration.processor.reprocess.minutes" should be in minutes.

Reply:
Fixed